adot/utils/expected-templates/python.json

```diff
@@ -9,7 +9,27 @@
     "name":"hello-lambda-python.*"
   },
   {
-    "name":"hello-lambda-python.*"
+    "name":"hello-lambda-python.*",
+    "subsegments": [
+      {
+          "name": "HTTP GET"
+      },
+      {
+          "name": "S3",
+          "aws": {
+            "operation": "ListBuckets"
+          },
+          "metadata": {
+            "default": {
+              "aws.service": "s3",
+              "rpc.service": "S3",
```

Comment:
Since X-Ray backend doesn't use these fields, I'd recommend not actually validating them

Reply:
Thanks for the call out! It makes great sense to me 🙂 I've made the update!